From af548f4167a174f64f09d149458dd950219972ac Mon Sep 17 00:00:00 2001
From: Jerome Jiang <jianj@google.com>
Date: Tue, 6 May 2025 10:58:05 -0400
Subject: [PATCH] Fix length of kFileMagic

Compiler warning from clang:

warning: initializer-string for character array is too long
array size is 8 but initializer has size 9
(including the null terminating character);

Change-Id: I571c73343a1f3e9594acedd8cf8365411e9cd054
---
 aom_dsp/grain_table.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/aom_dsp/grain_table.c b/aom_dsp/grain_table.c
index 3ca875658b..6c525d9953 100644
--- a/aom_dsp/grain_table.c
+++ b/aom_dsp/grain_table.c
@@ -36,7 +36,7 @@
 #include "aom_dsp/grain_table.h"
 #include "aom_mem/aom_mem.h"
 
-static const char kFileMagic[8] = "filmgrn1";
+static const char kFileMagic[8] = { 'f', 'i', 'l', 'm', 'g', 'r', 'n', '1' };
 
 static void grain_table_entry_read(FILE *file,
                                    struct aom_internal_error_info *error_info,
-- 
2.51.0.338.gd7d06c2dae-goog

